File: templates/python/logistics_network_phase_portrait.py
"""logistics_network_phase_portrait: 物流与网络相平面画像（phase-plane 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='logistics and network analysis: phase portrait'):
    return make_template_figure('phase_plane', seed=3411, title=title, domain='logistics and network analysis', topic='phase portrait')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/logistics_network_phase_portrait.m
function fig = logistics_network_phase_portrait()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('phase_plane', 3411, 'logistics and network analysis: phase portrait', 'logistics and network analysis', 'phase portrait');
end
